### ISSUE TITLE
Totems Awakening 2: Level 3 is impossible

### ISSUE THREAD

**Issue by n0samu** [ruffle-rs/ruffle#10383]
### Describe the bug

Link to SWF: http://i.notdoppler.com/files/totemsawakening2.swf

In level 3 of [Totems Awakening 2](http://i.notdoppler.com/files/totemsawakening2.swf), there is a barrier that you're supposed to be able to drag up and down. But in Ruffle, the barrier cannot be interacted with at all, making the level impossible.

### Expected behavior

It should be possible to drag the barrier up and down as the tutorial shows.
![totemsawakening2_level3](https://user-images.githubusercontent.com/71368227/227764954-a0eff3d5-cf25-4619-8e55-eec5edfbce2c.gif)

### Affected platform

Desktop app

### Operating system

Windows 10

**Comment by Lord-McSweeney** [comment]
I get this error when trying to start the SWF:
```
Error dispatching event EventObject(EventObject { type: "click", class: flash.events::MouseEvent, ptr: 0xd75c88 }) to handler FunctionObject(FunctionObject { ptr: 0x2318bb0 }) : TypeError: Error #1009: Cannot access a property or method of a null object reference. (accessing field: http://adobe.com/AS3/2006/builtin::indexOf)
	at GameSounds/initIDs()
	at GameSounds/init()
	at Main()
	at Preloader/startup()
	at Preloader/onPlayButtonClick()
```

**Comment by n0samu** [comment]
Yes that's the previous error with this game (#10086). It will be fixed in the upcoming nightly.

**Comment by n0samu** [comment]
Fixed by #10992
